packages/cli/src/ceramic-daemon.ts

```diff
@@ -57,6 +57,10 @@ const ADMIN_CODE_CACHE_CAPACITY = 50
 const STREAM_PINNED = 'stream_pinned'
 const STREAM_UNPINNED = 'stream_unpinned'
 const ERROR_QUERYING_COLLECTION = 'error_querying_collection'
+const METRIC_CREATE_STREAM_SPAN = 'http_request_create_stream_span'
```

Comment:
i like to name them exactly the same as their string, but that's just a personal preference, i think this is fine